docs/screens/pullrequests_screen.md

```diff
@@ -0,0 +1,37 @@
+# Pull Requests Screen
+
+<img src="../analytics_imgs/pull_requests/prs_screen_opened.png" width=20% height=20%>
+
+## Documentation
+
+|                           When pull requests screen opened.                            | Specification                                                                                   |
+|:--------------------------------------------------------------------------------------:|-------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/pull_requests/prs_screen_opened.png" width=50% height=50%> | <pre>{<br />   "name": "screen_opened"<br />   "item_name": "pull_requests_screen"<br />}<pre/> |
+
+|                           When user pulls to refresh.                            | Specification                                                                                        |
+|:--------------------------------------------------------------------------------:|------------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/pull_requests/refresh_prs.png" width=50% height=50%> | <pre>{<br />   "name": "action_start"<br />   "item_name": "refresh_pull_requests_data"<br />}<pre/> |
+
+|                         When refresh data finished with success.                         | Specification                                                                                                                   |
+|:----------------------------------------------------------------------------------------:|---------------------------------------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/pull_requests/refresh_prs_success.png" width=50% height=50%> | <pre>{<br />   "name": "action_finished"<br />   "item_name": "refresh_pull_requests_data"<br />   "success": true<br />}<pre/> |
+
+|                         When refresh data finished with failure.                         | Specification                                                                                                                    |
+|:----------------------------------------------------------------------------------------:|----------------------------------------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/pull_requests/refresh_prs_failure.png" width=50% height=50%> | <pre>{<br />   "name": "action_finished"<br />   "item_name": "refresh_pull_requests_data"<br />   "success": false<br />}<pre/> |
+
+|                     When user opens pull requests screen.                      | Specification                                                                                      |
```

Comment:
**chore**: "When user opens pull requests screen." then `screen_opened` event is called. `action_start` is called when data is started loading. I know this is usually the same moment, but not preciesly. If user get back to the screen data is not loaded, but screen is presented. So `screen_opened` is called, but `action_start` is not called.